src/utils/studies_functions.py

```diff
@@ -70,7 +70,7 @@ async def email(inviter: str, recipient: str, invitation_message: str, study_tit
         return response.status_code  # type: ignore
 
     except HTTPError as e:
-        logger.error(f"Email failed to send: {e}")
+        logger.error(f"Email failed to send: {e}", exc_info=True)
```

Comment:
Was `{e}` left in intentionally in this case?

Reply:
Ah yep! Since it's `logger.error()` still